test/e2e/density.go

```diff
@@ -187,8 +187,14 @@ var _ = PDescribe("Density", func() {
 	})
 
 	AfterEach(func() {
-		// Remove any remaining pods from this test
-		DeleteRC(c, ns, RCName)
+		// Remove any remaining pods from this test if the
+		// replication controller still exists and the replica count
+		// isn't 0.  This means the controller wasn't cleaned up
+		// during the test so clean it up here
+		rc, err := c.ReplicationControllers(ns).Get(RCName)
+		if err == nil && rc.Spec.Replicas != 0 {
```

Comment:
A retry might work but without a real world case to test against finding how many attempts and how long to wait between, or even if it will help, are all guesses.  I can add retries but I wouldn't be able to verify they'd actually solve a problem.  I'm also a little reluctant to cover up a communication problem.  The apiserver is going to have to be responsive under load.

I don't recall there being a lot of retries for operations in the e2e tests.  That is also probably because not many would stress the system to the point of causing communication timeouts like this test suite could.

ATM this test is disabled and won't be run unless explicitly enabled because of the nature of the test.  This really belongs in a performance test suite.